data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+	on complete
+		conversation
+			`Kirrie, already donning his winter coat, instructs you to land near the equator. He says him and Tugroob will conduct some subtle interrogations, stating that it's 'all they can do' in this climate.`
+			`	You drop them off in one of the larger settlements there. After they leave, Iplora asks that you bring her close to the southern pole of the planet.`
+			`	"Suspiciously similar, independent reports on that area have been. Tasked with investigating it, I was."`
+			`	As you land on an isolated icy patch in the middle of a snowstorm, Iplora finishes her preparations, carrying a bag with some equipment and wearing what is - at least when compared to what her colleagues were wearing - fairly light clothing. Although she is a Saryd, you still worry that she might be frostbitten in this extreme weather, but she reassures you that she will be fine.`
+			`	You open your ship's hatch and she prepares to leave.`
+			choice
+				`	"Good luck. Be careful not to get lost in all this snow."`
+					goto warm
+				`	"Hold up, I'll go with you so we can search faster."`
+			`	She pauses on the ramp, looking back at you.`
+			`	"Certain of that, are you, Captain? Treacherous, these storms can be."`
+			choice
+				`	"Don't worry, I can carry my weight through some snow."`
+					goto snow
+				`	"On second thought, you're right. Good luck out there."`
+			label warm
+			`	You slouch around your ship for a few hours, waiting for any signal from the Heliarch agents, but nothing comes up on your commlink. Iplora comes back, saying she couldn't find anything, so you leave and pick up Kirrie and Tugroob. They also return empty handed.`
+				accept
+			label snow
+			`	She remains silent, and walks back inside. You close the hatch and prepare yourself for what's to come with the warmest clothing you could find. You open the hatch once more, and the both of you head out into the white fields. Not long after leaving your ship, you can make out small, dark lines nearby, probably pine trees. It seems you landed close to a forest. Unfortunately, they don't do much to protect you from the snowfall.`
+			`	You continue deeper into the lifeless forest for what feels like hours, so long that your feet start feeling numb from the cold. After yet another turn, Iplora guides you out of the forest to snowy hills. The storm is not so terrible here, so you can faintly see cave entrances in some of the more distant hills. Looking back at the woods, you see that there's a small shack in the outskirts, not too far from where you are now.`
+			`	"Check the caverns, we must. More relentless, the storm will soon become," Iplora says.`
+			choice
+				`	"Lead the way."`
+					goto bumpyroad
+				`	"You go ahead. I'll see if I can find anything in that shack."`
+			`	She nods, and starts making her way uphill. You start walking to the shack, but just a couple steps into it you trip on something hard, unlike the soft, deep snow you've been in up until now.`
```

Comment:
Just FYI, right here, if your outer layer isn't waterproof - at this point you are now dead. Congratulations.

Reply:
Agreed.
But again, do I really need to specifically mention the player's winter clothing is waterproof?